### ISSUE TITLE
Expose all uWS websocket options

### ISSUE THREAD

**Issue by e3dio** [oven-sh/bun#2988]
### What is the problem this feature would solve?

Missing key websocket settings

### What is the feature you are proposing to solve the problem?

Expose all uWS websocket options:

Settings:
```
closeOnBackpressureLimit
idleTimeout
maxBackpressure
maxLifetime
maxPayloadLength
sendPingsAutomatically
```
Events:
```
ping
pong
subscription
```

**Comment by Jarred-Sumner** [comment]
These are exposed:

- maxPayloadLength
- idleTimeout
- backpressureLimit
- closeOnBackpressureLimit

**Comment by e3dio** [comment]
Ok just need to update the docs https://bun.sh/docs/api/websockets

Those are the main ones I wanted, the missing ones I don't need now:
- ping (event)
- pong (event)
- subscription (event)
- maxLifetime
- sendPingsAutomatically

**Comment by e3dio** [comment]
I tried: 
```js
websocket: {
   idleTimeout: 30
}
```
but does not seem to work. This should send the Ping automatically every 30 seconds to keep connection open. I need this to get past AWS load balancer 60 second timeout. Default websocket idleTimeout is 120 seconds. Tested with uWS.js `idleTimeout: 30` and it sends ping, keeps connection open

Confirmed bun `maxPayloadLength` working

**Comment by Jarred-Sumner** [comment]
Yeah it looks like pings aren't being sent. We are propagating the flag to uWS, but there's probably more we need to do

**Comment by DoKoB0512** [comment]
@Jarred-Sumner Any updates on this issue?
